src/map/status.cpp

```diff
@@ -9298,6 +9298,7 @@ int status_change_start(struct block_list* src, struct block_list* bl,enum sc_ty
 				return 0;
 			break;
 		case SC_STONE:
+		case SC_STONEWAIT:
```

Comment:
We also need to add SC_STONEWAIT in status_get_sc_def or else the resistance there won't work either.